java-core/solid/fruit-shop-tests.md

```diff
@@ -1,5 +1,6 @@
 ## Common mistakes (jv-fruit-shop-tests)
 
+##### Don't begin class or method implementation with empty line.  Remove all redundant empty lines, be careful :)
```

Comment:
```suggestion
#### Don't begin class or method implementation with empty line.  Remove all redundant empty lines, be careful :)
```
let's use `####` for now, because other points have such formatting